testing/test.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+import json
+import shutil
+import subprocess
+import shlex
+import argparse
+
+# Set up argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument(dest="path_to_exercise", metavar="PATH_TO_EXERCISE", type=str, help="the path to the exercise")
+args = parser.parse_args()
+
+path_to_exercise = args.path_to_exercise
+assert path_to_exercise 
+assert os.path.exists(path_to_exercise) 
+assert os.path.isdir(path_to_exercise)
+
+print("parsing config.json")
+
+exercise_config = open(os.path.join(path_to_exercise, "config.json"))
+assert exercise_config

```

Comment:
I think open will either return an open file handle, or throw an exception, but never return `None` or another Falsy value, so this assertion would never fail.